### PR TITLE
re-build post setuptools 61.0.0

### LINKED ISSUE #26: UI not bundled with UIS
### Solution to issue cannot be found in the documentation.

- [X] I checked the documentation.

### Issue

The bundled UI is not installed when cylc-uiserver is installed from conda-forge (version 1.0.1).

Things I've checked:

* [x] The packaging looks correct (compare against cylc-flow)
* [x] The UI is correctly included in the PyPi files (both the wheel and the src)
* [x] The setuptools version is not ==61.0.0 (because this has a known bug)

Demonstration of the issue in code.

This creates envs by installing cylc-uiserver from pypi or conda-forge then checking for the existence of the UI dir in the pip logs:

```bash
set -eux                                                                                      
                                                                                              
test_env () {                                                                                 
    ENV="$1"                                                                                  
    conda run -n "$ENV"                                                                       
    if grep \                                                                                 
        'ui\/' \                                                                              
        "$CYLC_HOME_ROOT_ALT/$ENV/lib/python"3.*"/site-packages/cylc_uiserver-1.0.1.dist-info/RECORD"
    then                                                                                      
        echo SUCCEEDED                                                                        
    else                                                                                      
        echo FAILED                                                                           
    fi                                                                                        
}                                                                                             
                                     
make_env_pypi_normal () {        
    local ENV='test-pypi-normal'    
    conda env remove -n "$ENV" || true    
    conda create -y -n "$ENV"    
    conda run -n "$ENV" conda install -y python=3.7 --offline    
    conda run -n "$ENV" pip install cylc-flow==8.0rc2    
    conda run -n "$ENV" pip install cylc-uiserver    
    test_env "$ENV"              
}                                             

make_env_conda () {              
    local ENV='test-conda'       
    conda env remove -n "$ENV" || true    
    conda create -y -n "$ENV"    
    conda run -n "$ENV" conda install -y python=3.7 cylc-flow==8.0rc2 cylc-uiserver --offline    
    test_env "$ENV"              
}                                
                                 
# this will give: SUCCEEDED      
make_env_pypi_normal             
# this will give: FAILED         
make_env_conda                   
```

### Installed packages

```shell
cylc-flow==8.0rc1
cylc-uiserver==1.0.1
```

### Environment info

```shell
N/A
```

### PR DESCRIPTION
**Pull request by oliver-sanders**
If successful this will close #26

tldr; setuptools 61.0.0 was broken in a way that could cause the `MANIFEST.in` file to be ignored. When we released the cylc-uiserver it looks like this version was used during the release. This may be the cause of the missing UI files.

If so all we need to to is release a new build (there have been two subsequent setuptools releases since 61.0.0 apparently so a newer version should be used). I've attempted to add a packaging test.

Checklist
* [ ] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by oliver-sanders** [comment]
It's using setuptools 61.2.0 now:

https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=482110&view=logs&j=656edd35-690f-5c53-9ba3-09c10d0bea97&t=e5c8ab1d-8ff9-5cae-b332-e15ae582ed2d&l=1375

The tests failed because it couldn't locate the test script, flying blind here, the conda-build docs don't outline how to use test scripts not bundled in the repo.

**Comment by oliver-sanders** [comment]
Wow, it worked!

https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=482112&view=logs&j=656edd35-690f-5c53-9ba3-09c10d0bea97&t=e5c8ab1d-8ff9-5cae-b332-e15ae582ed2d&l=3011

**Review by MetRonnie — APPROVED** [review]
Do we need to re-render?

**Comment by oliver-sanders** [comment]
I've posted this on the conda-forge gitter as it's likely other projects are impacted too.

**Review by wxtim — APPROVED** [review]
- [x] Reasonable
- [x] Nice test
- [x] Built
